### DISCUSSION TITLE
AllowCrimsonCreep broken

### DISCUSSION THREAD

**Discussion by D3RKDUDE** [Pryaxis/TShock#2980] [Q&A]
I have a Tshock server version 5.2.0.0 intensy and even though crimson creep is turned off in the config, crimson is still spreading very fast in front of our eyes.  Do you have a solution?

![InShot_20231005_234119814](https://github.com/Pryaxis/TShock/assets/145405605/0c41f85c-c272-41df-9b1d-a25c3c479248)

**Comment by ghost** [comment]
Try using journey mode...

**Reply by D3RKDUDE** [reply]
But what if i want to play master mode?

**Reply by ghost** [reply]
set the difficulty in the "Journey settings"

**Reply by D3RKDUDE** [reply]
So when i use journey settings, both world will be master like and also evil biomes stop spreading?

**Reply by ghost** [reply]
yes you also have to turn off the creeping through that as well
(dupe is allowed)

**Reply by D3RKDUDE** [reply]
Oh ok and that is dupe btw?

**Comment by ghost** [comment]
I meant that duplication is enabled if you set it to journey and master mode probably overrides it
________________________________
